### PR TITLE
New: JetBrains.MPS.EAP version 212.4416.671

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with winget validate --manifest <path>?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23833)

### CODE REVIEW

File: manifests/j/JetBrains/MPS/EAP/212.4416.671/JetBrains.MPS.EAP.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: JetBrains.MPS.EAP
+PackageVersion: 212.4416.671
+PackageLocale: en-US
+Publisher: JetBrains s.r.o
```
Comment (ItzLevvie) [inline manifests/j/JetBrains/MPS/EAP/212.4416.671/JetBrains.MPS.EAP.locale.en-US.yaml:7]:
```suggestion
Publisher: JetBrains
```

File: manifests/j/JetBrains/MPS/EAP/212.4416.671/JetBrains.MPS.EAP.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: JetBrains.MPS.EAP
+PackageVersion: 212.4416.671
```
Comment (ItzLevvie) [inline manifests/j/JetBrains/MPS/EAP/212.4416.671/JetBrains.MPS.EAP.locale.en-US.yaml:5]:
```suggestion
PackageVersion: MPS-212.4416.671
```

File: manifests/j/JetBrains/MPS/EAP/212.4416.671/JetBrains.MPS.EAP.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: JetBrains.MPS.EAP
+PackageVersion: 212.4416.671
```
Comment (ItzLevvie) [inline manifests/j/JetBrains/MPS/EAP/212.4416.671/JetBrains.MPS.EAP.installer.yaml:5]:
```suggestion
PackageVersion: MPS-212.4416.671
```

File: manifests/j/JetBrains/MPS/EAP/212.4416.671/JetBrains.MPS.EAP.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: JetBrains.MPS.EAP
+PackageVersion: 212.4416.671
```
Comment (ItzLevvie) [inline manifests/j/JetBrains/MPS/EAP/212.4416.671/JetBrains.MPS.EAP.yaml:5]:
```suggestion
PackageVersion: MPS-212.4416.671
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**